setup.cfg

```diff
@@ -2,12 +2,12 @@
 current_version = major
 
 [flake8]
-ignore = E731, E402
+ignore = E701, E731, E402, W504, C814
```

Comment:
E701 seems like a legit error to me, do we want to whitelist that?

Reply:
yep please use same settings as in raiden. This is one of the important things to have when having multiple projects